src/css/main.css

```diff
@@ -0,0 +1,167 @@
+.content {
+  min-height: 100vh;
+  background: linear-gradient(rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.4));
+               /*url(../assets/images/res5.jpg) center/cover no-repeat fixed;*/
+  margin-top: auto;
+  position: relative;
+}
+
+.collapse_button {
+  border: 1px solid pink;
+}
+
+.navbar-collapse {
+  flex-grow: 0;
+}
+
+.title {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-attachment: fixed;
+  width: 100%;
+  z-index: 2;
+  padding: 250px 0;
+  font-size: 50px;
+}
+
+.pulsate {
+  animation: pulsate 3s ease-in-out;
+  animation-iteration-count: infinite;
+  opacity: 0.3;
+}
+
+@-webkit-keyframes pulsate {
+  0% {
+    opacity: 0.3;
+  }
+
+  50% {
+    opacity: 1;
+  }
+
+  100% {
+    opacity: 0.3;
+  }
+}
+
+.category {
+  padding: 15px;
+}
+
+.categoryIcon svg {
+  width: 60px;
+  height: 60px;
+  margin-right: 20px;
+}
+
+.categoryIcon svg path {
+  fill: #ffc107;
+}
+
+.categoryIcon:hover,
+.cat_title:hover {
+  transition: transform 2s ease, color 2s ease;
+  transform: translateY(-10px);
+  color: #cd6100;
+  cursor: pointer;
+}
+
+.footerSection {
+  padding: 50px 0;
+}
+
+.footerAboutDiv,
+.jobsDiv {
+  justify-content: center;
+}
+
+.socialIcons a {
+  display: inline-block;
+  padding: 5px 10px;
+  margin: 0 2.5px;
+  width: 40px;
+  height: 40px;
+  border-radius: 5px;
+  transition: transform 2s ease, color 2s ease;
+}
+
+.socialIcons a:hover {
+  transform: translateY(-15px);
+}
+
+.dish {
+  background: linear-gradient(rgba(205, 96, 0, 0.7), rgba(205, 96, 0, 0.7));
+      /*url(../assets/images/res5.jpg) center/cover no-repeat fixed;*/
```

Comment:
This should work, have you tried it?

Reply:
Yes, but it didn't work at first. So I installed ```file-loader``` as recommended by you and it's now working.